docs/rst/fastdds_qos_profiles_manager_cli/cli_elements/cli_common_elements/cli_common_duration_type_element.rst

```diff
@@ -0,0 +1,45 @@
+.. _fastdds_qos_profiles_manager_cli_common_duration_type_element:
+
+Duration type configuration
+---------------------------
+
+This section explains how to access the different configurable parameters in a duration type configuration.
+The scope of this section does not include explaining the usage of these specific parameters.
+Please, refer to `Fast DDS documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/common.html#durationtype>`_ for that information.
+
+The duration type configuration is accessed through the ``duration`` element which is a complex parameter according to the :ref:`fastdds_qos_profiles_manager_element_classification`.
+The duration type configurable ``<subelements>`` are explained in the table below.
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - ``subelement``
+      - CLI access
+      - :ref:`fastdds_qos_profiles_manager_element_classification`
+      - Valid ``set`` values
+    * - Seconds
+      - ``duration.<duration_type>.sec``
+      - Simple parameter
+      - ``int32_t``
+    * - Nanoseconds
+      - ``duration.<duration_type>.nanosec``
+      - Simple parameter
+      - ``uint32_t``
+    * - Infinite duration type
+      - ``duration.<duration_type>``
+      - Complex parameter
```

Comment:
It has conflicts with our premise: "complex parameters are not final and another subelement can be configured within them"
May we say here that it works as a simple parameter?
```suggestion
      - Simple parameter
```

Reply:
I do not think there is a conflict. It is a complex parameter because seconds and nanoseconds can be configured. But some complex parameters are allowed to be configured (and those are the ones that we need to document clearly)